docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,136 @@
+.. warning::
+    This is alpha functionality. While usable, this has not yet been tested in a large
+    number of different scenarios. The API may also still be subject to change.
+
+Piezo tracking
+==============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+In this tutorial, we will determine a high frequency distance (piezo distance) between the beads from the piezo mirror position of trap 1 and the measured force data.
+We will also show how to use the same reference curve to apply baseline correction for force signals in post processing.
+
+Positional Calibration
+----------------------
+
+The first step is to perform a positional calibration to calibrate the trap position to the distance measured by the camera.
+
+To do this calibration, we require a dataset acquired in the absence of a tether.
+In this dataset, trap 1 is moved over range, while the trap 1 position and camera-based distance is recorded.
+
+Note that this dataset should be acquired such that the bead tracking templates do not overlap.
+In case of overlap, make sure to slice the position and distance data such that this region is not included.
+Let's load this dataset and perform the distance calibration by invoking::
```

Comment:
I think here it's nice to add some emphasis to this caveat and to separate it from the final line, since they are two different thoughts

```suggestion
*Note that this dataset should be measured such that the bead tracking templates do not overlap. In case of overlap, make sure to first slice the position and distance data such that this region is not included.*

Let's load this dataset and perform the distance calibration by invoking::
```